packages/editor/src/components/ComponentsList/ComponentList.tsx

```diff
@@ -80,15 +156,74 @@ export const ComponentList: React.FC<Props> = ({ services }) => {
       })),
       c => -getCategoryOrder(c.name)
     );
-  }, [filterText, registry]);
+  }, [filterText, registry, checkedVersions]);
 
   return (
     <>
-      <Input
-        placeholder="filter the components"
-        value={filterText}
-        onChange={evt => setFilterText(evt.currentTarget.value)}
-      />
+      <InputGroup>
+        <Input
+          placeholder="filter the components"
+          value={filterText}
+          onChange={evt => setFilterText(evt.currentTarget.value)}
+        />
+        <InputRightElement>
+          <Popover closeOnBlur placement="bottom">
```

Comment:
The popover should be wrappered to a new component. It is too chaos to mix with `ComponentList`

Reply:
done